.github/ISSUE_TEMPLATE.md

```diff
@@ -3,11 +3,11 @@ Before submitting, please read over our commonly reported issues to prevent dupl
 
 ### All common issues
 
-* [common issues](/docs/common-issues)
+* [common issues](/docs/common-issues.md)
```

Comment:
@ljharb Isnt it ok to be absolute links on github as this is an issue template which shows up in github when you raise an issue.

Or do we want to show the links instead of markdown style?